builtInFunctions/esdtTransfer.go

```diff
@@ -183,7 +200,15 @@ func (e *esdtTransfer) ProcessBuiltinFunction(
 			vmOutput)
 	}
 
-	addESDTEntryInVMOutput(vmOutput, []byte(core.BuiltInFunctionESDTTransfer), tokenID, 0, value, vmInput.CallerAddr, vmInput.RecipientAddr)
+	addESDTEntryForTransferInVMOutput(
```

Comment:
some unit tests that would have ensured the log data is still the same after using the new function would have helped

Reply:
VM has a lot of mandos tests that check the logs for builtin functions